### ISSUE TITLE
Support annotations in service

### ISSUE THREAD

**Issue by M-Nawar** [qdrant/qdrant-helm#15]

**Comment by sebastian-montero** [comment]
Thank you @M-Nawar  for opening this issue. For context, I am trying to deploy a service using the `LoadBalancer` service type, however this should be accesible only within my GCP project. I am trying to include the following:
```
service:
  type: LoadBalancer
  annotations:
    networking.gke.io/load-balancer-type: "Internal"
```

**Comment by M-Nawar** [comment]
Hi @sebastian-montero  this issue is now resolved in qdrant-helm 0.2.6, you can now run the installation command with
--set service.type=LoadBalancer --set service.annotations."networking\.gke\.io/load-balancer-type"=Internal
to support your usecase
